CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.6.2)
```

Comment:
Should this stay at 3.6.0? Or is it good practice to keep this the same version as CI? When I commented that CI could use the latest version, I assumed without thinking that these could stay 3.6.0, since it seems unnecessary for force users to a specific minor release number.